app/views/applications_mailer/approved.html.haml

```diff
@@ -3,17 +3,19 @@
     %td
       %p Hi #{@user.name}!
 
-      %p We'd like to offer you membership to Double Union! YAY! There's so much to do now that you're a member :D
+      %p We'd like to offer you membership to Double Union! YAY!
 
-      %p As soon as possible, log into #{ link_to "the site", EXTERNAL_SITE_URL } ("Log in" is at top right) and confirm your membership #{ link_to "on this page", members_user_setup_url(@user.id) }. This is where you'll give us your Google Drive-friendly email address, set up dues (or apply for a scholarship), and check your subscription to the members' mailing list. (A membership coordinator will subscribe you to the mailing list soon. Feel free to start a thread introducing yourself and jump into conversations right away.)
+      %p In order to confirm your membership, log into #{ link_to "the site", EXTERNAL_SITE_URL } ("Log in" is at top right) and complete the setup #{ link_to "on this page", members_user_setup_url(@user.id) }. This is where you'll give us your Google-friendly email address for our calendar and mailing list, and set up dues (or apply for a scholarship), and check your subscription to the members' mailing list. (After you’ve done this, a membership coordinator will subscribe you to the mailing list soon. Feel free to start a thread introducing yourself and jump into conversations right away!)
```

Comment:
Also, do we want to say anything about adding them to the DU Slack, since that seems about as active as the mailing list these days?